test/unit/js/ASpellWorkerTests.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable
+    handle-callback-err,
+    no-undef
+*/
+// TODO: This file was created by bulk-decaffeinate.
+// Sanity-check the conversion and remove this comment.
+/*
+ * decaffeinate suggestions:
+ * DS102: Remove unnecessary code created because of implicit returns
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
```

Comment:
Slightly odd to see a new file pop up with the decaf mumbojumbo in it, but I'll allow it. :-)